homeassistant/components/jellyfin/coordinator.py

```diff
@@ -52,18 +52,21 @@ async def _async_update_data(self) -> JellyfinDataT:
     @abstractmethod
     async def _fetch_data(self) -> JellyfinDataT:
         """Fetch the actual data."""
-        raise NotImplementedError
 
 
 class SessionsDataUpdateCoordinator(
     JellyfinDataUpdateCoordinator[dict[str, dict[str, Any]]]
 ):
     """Sessions update coordinator for Jellyfin."""
 
-    async def _fetch_data(self) -> dict:
+    async def _fetch_data(self) -> dict[str, Any]:
```

Comment:
This is the type of the coordinator above.
```suggestion
    async def _fetch_data(self) -> dict[str, dict[str, Any]]:
```